Linphone/Core/ICoreHttpClient.cs

```diff
@@ -0,0 +1,27 @@
+﻿using BelledonneCommunications.Linphone.Presentation.Dto;
+using System;
+using System.Threading.Tasks;
+
+namespace BelledonneCommunications.Linphone.Core
+{
+    public interface ICoreHttpClient
```

Comment:
Please move all this code to the data leyer, like Mobayyen's hardware project architecture (please see directory structure and layering model)

Reply:
Resolved.